package.json

```
@@ -4,8 +4,9 @@
   "scripts": {
     "type-check": "tsc",
     "generate": "npm-run-all generate:*",
-    "generate:payment-ecommerce-webview": "rimraf generated/definitions/payment-ecommerce-webview && shx mkdir -p generated/definitions/payment-ecommerce-webview && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/v1.292.0/src/domains/ecommerce-app/api/ecommerce-webview/v1/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-webview --request-types --response-decoders --client",
-    "generate:payment-ecommerce": "rimraf generated/definitions/payment-ecommerce && shx mkdir -p generated/definitions/payment-ecommerce && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/v1.292.0/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce --request-types --response-decoders --client",
+    "generate:payment-ecommerce-webview-v2": "rimraf generated/definitions/payment-ecommerce-webview-v2 && shx mkdir -p generated/definitions/payment-ecommerce-webview-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-3613-update-get-transaction-definition/src/domains/ecommerce-app/api/ecommerce-webview/v2/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-webview-v2 --request-types --response-decoders --client",
```

```suggestion
    "generate:payment-ecommerce-webview-v2": "rimraf generated/definitions/payment-ecommerce-webview-v2 && shx mkdir -p generated/definitions/payment-ecommerce-webview-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/CHK-3613-update-get-transaction-definition/src/domains/ecommerce-app/api/ecommerce-webview/v2/_openapi.json.tpl --no-strict --out-dir ./generated/definitions/payment-ecommerce-webview-v2 --request-types --response-decoders --client",
```

TODO branch ref to be updated with tag version once dependent infra PR merge